benchmarks/pytorch_benchmark_rmsnorm.py

```diff
@@ -0,0 +1,307 @@
+import os
+
+import matplotlib.pyplot as plt
+import numpy as np
+
+import torch
+import torch.nn as nn
+
+from quack.rmsnorm import QuackRMSNorm
+from tabulate import tabulate
+
+
+class RMSNorm(nn.Module):
+    def __init__(self, dim, eps=1e-5):
+        super().__init__()
+        self.eps = eps
+        self.scale = nn.Parameter(torch.ones(dim))
+
+    def forward(self, x):
+        norm_x = x.norm(2, dim=-1, keepdim=True)
+        rms_x = norm_x * torch.rsqrt(torch.tensor(x.shape[-1], dtype=x.dtype))
+        x_normed = x / (rms_x + self.eps)
+        return self.scale * x_normed
+
+
+def benchmark_implementation(
```

Comment:
i generally prefer triton.testing.do_bench since it clears the L2 cache as well

Reply:
yes, was just thinking about this!  Let me update to that.